ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -785,14 +786,15 @@ static KsqlRestApplication buildApplication(
     final Admin internalAdmin = createCommandTopicAdminClient(restConfig, ksqlConfig);
     final KafkaTopicClient internalTopicClient = new KafkaTopicClientImpl(() -> internalAdmin);
 
+    final AppInfoParser.AppInfo appInfo = new AppInfoParser.AppInfo(System.currentTimeMillis());
```

Comment:
Wouldn't this be a different app info result than what's in KsqlConfig since the system time would be different?

Reply:
Yes, it was being created new with each call anyways. And the call in ksqlConfig was generating like 95% of our log traffic in soak

Reply:
Yes, it was being created new with each call anyways. It only records the commit and version so I don't think that should change. Also the call in ksqlConfig was generating like 95% of our log traffic in soak and it was added since the last release

@stevenpyzhang